frontend/src/stack/create/create-by-form.html

```diff
@@ -10,7 +10,7 @@
             <div class="form-control">
                 <input id="createByForm-Namespace" type="text" data-ng-model="stackCreateByFormCtrl.form.Namespace"
                        data-required="required" name="Name" focus-me
-                       data-ng-pattern="/^[A-Za-z0-9]+$/">
+                       data-ng-pattern="/^\w[\w_.-]+\w$/">
```

Comment:
If we don't want character _ to appear in stack namespace, the pattern should be "/^[a-zA-Z0-9][a-zA-Z0-9.-]*[a-zA-Z0-9]$/".